pkg/login/social/social.go

```diff
@@ -223,3 +240,27 @@ var GetOAuthProviders = func(cfg *setting.Cfg) map[string]bool {
 
 	return result
 }
+
+func readConfig(configFile string) (*setting.OAuthGroupMappingsConfig, error) {
+	result := &setting.OAuthGroupMappingsConfig{}
+
+	oauthLogger.Debug("Reading group mapping file", "file", configFile)
+
+	_, err := toml.DecodeFile(configFile, result)
+	if err != nil {
+		return nil, errutil.Wrap("Failed to load OAuth group mappings file", err)
```

Comment:
Error strings should not be [capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings):
```suggestion
		return nil, errutil.Wrap("failed to load OAuth group mappings file", err)
```